aws/table_aws_ecs_container_instance.go

```diff
@@ -0,0 +1,226 @@
+package aws
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsEcsContainerInstance(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ecs_container",
+		Description: "AWS ECS Container",
+		List: &plugin.ListConfig{
+			ParentHydrate: listEcsClusters,
+			Hydrate:       listEcsContainerInstances,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "container_instance_arn",
+				Description: "The namespace Amazon Resource Name (ARN) of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ContainerInstanceArn"),
+			},
+			{
+				Name:        "instance_id",
```

Comment:
```suggestion
				Name:        "ec2_instance_id",
```

I think this column name should remain `ec2_instance_id`, to help differentiate as the EC2 instance ID vs. the container instance ID